app/entry.server.tsx

```diff
@@ -97,9 +98,16 @@ export function handleError(
 	error: unknown,
 	{ request }: LoaderFunctionArgs | ActionFunctionArgs,
 ): void {
+	// Skip capturing if the request is aborted as Remix docs suggest
```

Comment:
This is taken directly from Sentry, so the Sentry behavior won't change:

https://github.com/getsentry/sentry-javascript/blob/531779300c186f89afff0c5bad9f802b2140a325/packages/remix/src/utils/instrumentServer.ts#L151-L156